.github/workflows/trigger-release.yml

```
@@ -0,0 +1,36 @@
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        required: false
+        description: 'Version number for the release'
+
+name: Manually trigger a release
+
+jobs:
+  release:
+    name: Trigger a release
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      INPUT_VERSION: ${{ github.event.inputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

May as well update these to the latest versions. Reading their changelogs, it looks like the breaking change between v2 and v4 is that they updated the version of Node.js these actions use internally.
```suggestion
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
```